pylint/checkers/base.py

```diff
@@ -1630,8 +1641,13 @@ def _check_docstring(self, node_type, node, report_missing=True,
                         return
                     if func.bound.name in ('str', 'unicode', 'bytes'):
                         return
-            self.add_message('missing-docstring', node=node, args=(node_type,),
-                             confidence=confidence)
+            if node_type == 'module':
```

Comment:
Can you add some functional tests for these new messages? I would also expect some tests that enables and disables the new messages in pair with the old messages, something along the lines of:
```
# pylint: disable=missing-function-docstring
def func():
    ...

# pylint: eanble=missing-function-docstring
def other_func(): # now it should emit here
```

Reply:
Done in latest commit.